streams/src/main/java/org/apache/kafka/streams/StreamsBuilder.java

```diff
@@ -974,6 +974,7 @@ public synchronized StreamsBuilder addGlobalStore(final StateStoreSupplier<KeyVa
      * @param streams the {@link KStream}s to be merged
      * @return a {@link KStream} containing all records of the given streams
      */
+    @Deprecated
```

Comment:
Please add `@deprected use {@link KStream#merge(KStream stream)} instead` to method JavaDoc